scripts/tests/chiptest/test_definition.py

```diff
@@ -154,6 +163,9 @@ class ApplicationPaths:
     lock_app: typing.List[str]
     tv_app: typing.List[str]
 
+    def get(self):
```

Comment:
use all/items/list or similar to make it clear what it does. Get is unclear on what it would get.